src/cli/argument_default_value_from_config_provider.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Optional
+
+from src.cli.command import Command
+from src.utils import Project
+
+
+class ArgumentDefaultValueFromConfigProvider:
```

Comment:
We we use `Default` in the name here? I think it's a bit misleading in connection with python defaults.
`ArgumentValueFromConfigProvider` would be better

Reply:
The config file changes the default value of argument. However, this class shouldn't care how it's used, therefore I'll rename it.